ratelimit.gemspec

```diff
@@ -18,9 +18,9 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency             "redis", ">= 2.0.0"
+  spec.add_dependency             "redis", ">= 3.0.0"
   spec.add_dependency             "redis-namespace", ">= 1.0.0"
-  spec.add_development_dependency "bundler", "~> 1.6"
+  spec.add_development_dependency "bundler", ">= 1.6"
```

Comment:
Added this, so the project would also run on newer versions of ruby (with bundler > 2.0). I can also remove this again if you wish.